modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/LinkOrButton.js

```diff
@@ -53,6 +55,7 @@ const LinkOrButton = React.forwardRef(
 						className={classNames(className, 'd-md-flex d-none', {
 							'pl-4 pr-4': wide,
 						})}
+						data-qa-id={`${dataQaId}Responsive`}
```

Comment:
Since you are changing the value of the qa id here, make sure to search for any usages that may need to be updated.